app/model/Event/ISkautisEvent.php

```diff
@@ -4,9 +4,6 @@
 
 namespace Model\Skautis;
 
-use Model\Common\UnitId;
-
 interface ISkautisEvent
 {
-    public function getUnitId(): UnitId;
```

Comment:
Tohle jsem musel odebrat, abych od toho mohl odvodit `SkautisEducationCourse` - alternativně ho nebudu odvozovat a neodeberu odsud to `UnitId`...